src/ziggurat/streams.clj

```diff
@@ -2,29 +2,26 @@
   (:require [clojure.tools.logging :as log]
             [mount.core :as mount :refer [defstate]]
             [ziggurat.channel :as chl]
-            [ziggurat.config :refer [ziggurat-config]]
+            [ziggurat.config :refer [get-in-config ziggurat-config build-properties]]
             [ziggurat.header-transformer :as header-transformer]
-            [ziggurat.message-payload :refer [->MessagePayload]]
             [ziggurat.mapper :refer [mapper-func]]
-            [ziggurat.util.logging :as zl]
+            [ziggurat.message-payload :refer [->MessagePayload]]
             [ziggurat.metrics :as metrics]
             [ziggurat.timestamp-transformer :as timestamp-transformer]
-            [ziggurat.util.map :as umap]
             [ziggurat.tracer :refer [tracer]]
-            [ziggurat.config :as config])
-  (:import [java.util Properties]
+            [ziggurat.util.map :as umap])
+  (:import [io.opentracing.contrib.kafka TracingKafkaUtils]
+           [io.opentracing.contrib.kafka.streams TracingKafkaClientSupplier]
+           [io.opentracing.tag Tags]
+           [java.time Duration]
+           [java.util Properties]
            [java.util.regex Pattern]
-           [org.apache.kafka.clients.consumer ConsumerConfig]
            [org.apache.kafka.common.serialization Serdes]
            [org.apache.kafka.common.utils SystemTime]
            [org.apache.kafka.streams KafkaStreams KafkaStreams$State StreamsConfig StreamsBuilder Topology]
            [org.apache.kafka.streams.kstream JoinWindows ValueMapper TransformerSupplier ValueJoiner ValueTransformerSupplier]
            [org.apache.kafka.streams.state.internals KeyValueStoreBuilder RocksDbKeyValueBytesStoreSupplier]
-           [ziggurat.timestamp_transformer IngestionTimeExtractor]
-           [io.opentracing.contrib.kafka.streams TracingKafkaClientSupplier]
-           [io.opentracing.contrib.kafka TracingKafkaUtils]
-           [io.opentracing.tag Tags]
-           [java.time Duration]))
+           [ziggurat.timestamp_transformer IngestionTimeExtractor]))
```

Comment:
I'm putting one comment for L33 here

we should change the value of session-timeout-ms-config to 60000 (which would same as what we have for consumer namespace)

Reply:
done